sparkle/src/components/Popover.tsx

```diff
@@ -0,0 +1,67 @@
+import * as PopoverPrimitive from "@radix-ui/react-popover";
+import * as React from "react";
+
+import { classNames } from "@sparkle/lib/utils";
+
+const PopoverRoot = PopoverPrimitive.Root;
+
+const PopoverTrigger = PopoverPrimitive.Trigger;
+
+interface PopoverContentProps
+  extends React.ComponentPropsWithoutRef<typeof PopoverPrimitive.Content> {
+  fullWidth?: boolean;
+}
+
+const PopoverContent = React.forwardRef<
+  React.ElementRef<typeof PopoverPrimitive.Content>,
+  PopoverContentProps
+>(
+  (
+    {
+      className,
+      align = "center",
+      sideOffset = 4,
+      fullWidth = false,
+      ...props
+    },
+    ref
+  ) => (
+    <PopoverPrimitive.Portal>
+      <PopoverPrimitive.Content
+        ref={ref}
+        align={align}
+        sideOffset={sideOffset}
+        className={classNames(
+          "data-[state=open]:s-animate-in data-[state=open]:s-fade-in-0 data-[state=open]:s-zoom-in-95",
+          "data-[state=closed]:s-animate-out data-[state=closed]:s-fade-out-0 data-[state=closed]:s-zoom-out-95",
+          "data-[side=bottom]:s-slide-in-from-top-2",
+          "data-[side=left]:s-slide-in-from-right-2",
+          "data-[side=right]:s-slide-in-from-left-2",
+          "data-[side=top]:s-slide-in-from-bottom-2",
+          "s-border-border s-text-primary-950 s-z-50 s-rounded-lg s-border s-bg-background s-shadow-md s-outline-none",
+          fullWidth ? "s-grow" : "s-w-72 s-p-4",
+          className ?? ""
```

Comment:
If you use `cn()` you can do:
```suggestion
          className
```